### DISCUSSION TITLE
gradle command doesn't work

### DISCUSSION THREAD

**Discussion by Sneezy123** [LanternPowered/Terre#5] [General]
if I write `gradle build` in my cmd on windows the command doesn't gets found. why?

**Comment by Cybermaxke** [comment]
`gradle` is a tool that can be installed on computer, if you don't have it installed you'll need to use `./gradlew` on linux or `./gradlew.bat` as a replacement for `gradle`. For example for the build command: `./gradlew build` or `./gradlew.bat build`. Make sure that the commandline is located in the project directory.

As a side node, I've already published some jars so you don't need to build it yourself, you can follow the steps in the wiki: https://github.com/LanternPowered/Terre/wiki/Getting-started
I'll add more info about configuring soon
